components/Gallery/index.js

```diff
@@ -1,26 +1,40 @@
-//Import external resources
+// Import external resources
 import styled from "styled-components";
+import useSWR from "swr";
 
-//Import internal resources
-import products from "@/products.json";
+// Import internal resources
 import ItemCard from "../ItemCard";
 
 // Import styles
 import * as Styled from "@/components/Gallery/Gallery.styles.js";
 
 export default function Gallery({ onhandleCartTotal }) {
+  const { data: products, isLoading, error } = useSWR("/api/products");
+
+  if (error) {
+    return (
+      <li>
+        <h2>{error.message}</h2>
+      </li>
```

Comment:
A list item should always be the child of a list and why do you wrap your heading with a list item in the first place?

You should remove the wrapper:

```suggestion
        <h2>{error.message}</h2>
```

Reply:
- [x] Removed li